dataland-frontend/src/utils/PremadeDropdownDatasets.ts

```diff
@@ -27,6 +27,17 @@ export function getDataset(datasetIdentifier: DropdownDatasetIdentifier): Dropdo
   throw Error(`Unknown dataset identifier ${datasetIdentifier as string}`);
 }
 
+/**
```

Comment:
The variable name mapOfCountries is a bit too specific for a dataset that can also hold different data.

Reply:
Something like mapOfDropdownOptions?

Reply:
Gern